syntax-testcases/functor1.ml

```diff
@@ -0,0 +1,19 @@
+module M : sig
+  module type F
+    = functor () (X : Ord) -> sig end (* not ok *)
```

Comment:
This comment and the other ones in this file refer to linting as it was done *before* PR #95. Now all these tests work.

Reply:
Some of the equal signs seem to not be highlighted, but I guess that's a minor issue.